src/Application/View/Extension/PluginExtension.php

```diff
@@ -85,4 +86,21 @@ public function callPluginHookFunction(string $name, $params = null, bool $retur
             return $result;
         }
     }
+
+    /**
+     * Call Plugin::getWebDir() with given params.
+     *
+     * @param string  $plugin
+     * @param bool    $full
+     * @param bool    $use_url_base
+     *
+     * @return mixed|void
+     */
+    public function getPluginWebDir(
+        string $plugin,
+        bool $full = true,
+        bool $use_url_base = false
+    ) {
+        return Plugin::getWebDir($plugin, $full, $use_url_base);
```

Comment:
I quickly read the `Plugin::getWebDir()` and the `Plugin::`Plugin::getWebDir()` ()` code and it appears that it does not handle correctly the value returned for a plugin that is not present on disk.

However, a `null` value should be returned here, if `Plugin::getWebDir()` does not return a string.